Spokestack/TextToSpeech.swift

```diff
@@ -37,6 +41,7 @@ import CryptoKit
     private lazy var player: AVPlayer = AVPlayer()
     private var apiKey: SymmetricKey?
     private let ttsInputVoices = [0: "demo-male"]
+    private let decoder = JSONDecoder()
```

Comment:
Is `decoder` as a class property to avoid allocating during every `synthesis` response?

Reply:
yes